src/index.ts

```diff
@@ -1,24 +1,30 @@
-import { createMachine, createActor, assign } from "xstate";
+import { createMachine, createActor, assign, raise } from "xstate";
 import { speechstate, Settings, Hypothesis } from "speechstate";
 
 const azureCredentials = {
   endpoint:
     "https://northeurope.api.cognitive.microsoft.com/sts/v1.0/issuetoken",
-  key: "",
+  key: "994d7aefd0894deca17e2b24dc6b7daf",
 };
 
+
 const settings: Settings = {
   azureCredentials: azureCredentials,
   asrDefaultCompleteTimeout: 0,
   locale: "en-US",
   asrDefaultNoInputTimeout: 5000,
   ttsDefaultVoice: "en-GB-RyanNeural",
 };
-
+// air-conditioner functions 
 interface DMContext {
   spstRef?: any;
   lastResult?: Hypothesis[];
-}
+  Mode?: string; 
```

Comment:
minor: by convention "variables" (not strictly variables, but fields of the context) shd be lower-cased